time_capture/time_capture/doctype/bulk_leave_application/bulk_leave_application.json

```
@@ -0,0 +1,245 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2025-04-09 14:17:35.501172",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "employee",
+  "employee_name",
+  "column_break_ihkc",
+  "leave_type",
+  "company",
+  "department",
+  "dates_section",
+  "table_leaves",
```

```suggestion
  "leaves",
```
The prefix `table_` does not follow Frappe's standard design principles.